packages/grafana-e2e/src/flows/login.ts

```diff
@@ -13,11 +13,10 @@ export const login = (username: string = 'admin', password: string = 'admin') =>
   e2e()
     .url()
     .then(url => {
-      if (/^https?:\/\/localhost/.test(url)) {
-        e2e.pages.Login.skip()
-          .should('be.visible')
-          .click();
-      }
+      // TODO: Detect if we're on the page for setting a new password
```

Comment:
I assume we need to update the condition to match env's HOST variable? guess we can do this with `e2e.env` method

Reply:
@dprokop Might it work instead to test if we're on the page for setting a new password? It would be more direct than assuming that we are on this page, if on a certain host.

Reply:
Yes, you are right. IMO the assertion below:
```
e2e.pages.Login.skip()
        .should('be.visible')
```

already covers the check if we are on the skip password page. Not sure what was the motivation for the host check.

Reply:
@dprokop Thanks, great that it's actually this simple. Then I can just drop the TODO comment :)